sysusers.d/basic.conf.in

```diff
@@ -5,9 +5,6 @@
 #  the Free Software Foundation; either version 2.1 of the License, or
 #  (at your option) any later version.
 
-# The superuser
-u root    0     "Super User" /root
```

Comment:
hmm, I think this is quite problematic, since people invoke sysusers offline all the time, and expect this to be created. sysusers is quite universally adopted but firstboot is not, hence dropping this is probably going to trip up quite a lot of people

Reply:
also, why drop this at all? i mean, sysusers should create the root user if missing with an unset password (if none is provided via credentials), so that all logins are refused, and firstboot should then do its thing on the condition the root pw is still invalid like that.